rllib/agents/dqn/apex.py

```diff
@@ -128,6 +148,54 @@ def __call__(self, item: Tuple[ActorHandle, SampleBatchType]):
 
 
 class ApexTrainer(DQNTrainer):
+    @override(Trainable)
+    def setup(self, config: PartialTrainerConfigDict):
+        super().setup(config)
+        num_replay_buffer_shards = self.config["optimizer"]["num_replay_buffer_shards"]
+
```

Comment:
Let's add an if-block here for _disable_execution_plan=True, such that for the `execution_plan` version of APEX, we don't create the learner thread and the actors twice.